packages/core/saved-objects/docs/openapi/components/examples/import_objects_response.yaml

```diff
@@ -0,0 +1,17 @@
+summary: The import objects API response indicates a successful import and the objects are created. Since these objects are created as new copies, each entry in the successResults array includes a destinationId attribute.
+value:
+  {
+    "successCount":1,
+    "success":true,
+    "warnings":[],
+    "successResults":[
```

Comment:
Our existing docs include `errors` in the [response](https://www.elastic.co/guide/en/kibana/current/saved-objects-api-import.html#saved-objects-api-import-response-body). Can we also give an example of the error case?

Reply:
I think it probably is reasonable to include https://www.elastic.co/guide/en/kibana/current/saved-objects-api-import.html#saved-objects-api-import-example-4 in a future update to this spec.

IMO https://www.elastic.co/guide/en/kibana/current/saved-objects-api-import.html#saved-objects-api-import-example-3 is probably best left in a verbose troubleshooting page, since the amount of explanation required seems a lot for a simple spec.

Reply:
That makes sense and we can add it later.